src/tests/functional/shared_test_classes/src/subgraph/input_conv_concat.cpp

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/subgraph/input_conv_concat.hpp"
+
+#include "ngraph_functions/builders.hpp"
+
+namespace SubgraphTestsDefinitions {
+
+using namespace CommonTestUtils;
+using namespace ngraph::builder;
+using namespace ov;
+using namespace ov::element;
+using namespace ov::opset10;
+using namespace std;
+
+string InputConvConcatTest::getTestCaseName(const testing::TestParamInfo<InputConvConcatParams>& obj) {
+    InferenceEngine::Precision net_precision;
+    vector<size_t> input_shapes;
+    string target_device;
+    map<string, string> configuration;
+    vector<size_t> kernel_shape;
+    size_t input_channels;
+    size_t output_channels;
+    tie(net_precision, input_shapes, target_device, configuration, kernel_shape, input_channels, output_channels) =
+        obj.param;
+
+    ostringstream result;
+    result << "PR=" << net_precision.name() << "_";
+    result << "IS=" << vec2str(input_shapes) << "_";
+    result << "KS=" << vec2str(kernel_shape) << "_";
+    result << "IC=" << input_channels << "_";
+    result << "OC=" << output_channels << "_";
+    result << "TD=" << target_device;
+    for (auto const& configItem : configuration) {
+        result << "_CI=" << configItem.first << "_" << configItem.second;
+    }
+    return result.str();
+}
+
+InferenceEngine::Blob::Ptr InputConvConcatTest::GenerateInput(const InferenceEngine::InputInfo& info) const {
+    InferenceEngine::Blob::Ptr blob = make_blob_with_precision(info.getTensorDesc());
+    blob->allocate();
+
+    auto* rawBlobDataPtr = blob->buffer().as<float*>();
+    vector<float> values = generate_float_numbers(blob->size(), -0.2f, 0.2f);
+    for (size_t i = 0; i < blob->size(); i++) {
+        rawBlobDataPtr[i] = values[i];
+    }
+    return blob;
+}
+
+shared_ptr<FakeQuantize> create_fq_node(const Type& type,
+                                        const shared_ptr<ov::Node>& node,
+                                        float fqMin,
+                                        float fqMax,
+                                        size_t levels) {
+    auto fqInpMin = makeConstant<float>(type, {1}, {fqMin});
+    auto fqInpMax = makeConstant<float>(type, {1}, {fqMax});
+    auto fqOutMin = makeConstant<float>(type, {1}, {fqMin});
+    auto fqOutMax = makeConstant<float>(type, {1}, {fqMax});
+    return make_shared<FakeQuantize>(node, fqInpMin, fqInpMax, fqOutMin, fqOutMax, levels);
+}
+
+void InputConvConcatTest::SetUp() {
```

Comment:
A few words of comment explaining what we're trying to test here would be welcome.

Reply:
Added short description.